File: problema07/sol_java/Ejercicio07/src/ejercicio07/Ejercicio07.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package ejercicio07;

import java.util.Scanner;

/**
 *
 * @author DETPC
 */
public class Ejercicio07 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        Scanner scanner = new Scanner(System.in);
        int numero = 0;
        int auxiliar = 0;
        int suma = 0;
        System.out.println("Ingrese un número par");
        numero = scanner.nextInt();
        auxiliar = numero;
        if (numero < 2 || numero % 2 != 0) {
            System.out.println("Error. El número debe ser mayor o igual a "
                    + "2 y par");
        } else {
            while (numero >= 2) {
                suma += numero;
                numero -= 2;
            }
            System.out.println("La suma de los números pares del 2 hasta el número " + auxiliar + " es: " + suma);
        }

    }

}

File: problema07/sol_python/Ejercicio07.py
suma=0
mensaje=""
contador=0
numero=int(input("Ingrese un número par: "))
if numero<2 or numero%2!=0:
    print("Error. El número debe ser mayor o igual a 2 y par. Vuelva a ingresar")

while contador<numero:
    contador=contador+2
    mensaje=mensaje+"+"+str(contador)
    suma=suma+contador
print(mensaje)
print("La suma es: ",suma)